DigitalLearningSolutions.Web/ViewModels/Common/SearchablePage/BaseSearchablePageViewModel.cs

```diff
@@ -10,12 +10,12 @@ public abstract class BaseSearchablePageViewModel : BasePaginatedViewModel
         public const string Descending = "Descending";
         public const string Ascending = "Ascending";
 
-        public readonly string? FilterBy;
-
         public readonly bool FilterEnabled;
 
         public readonly string? SearchString;
 
+        public string? FilterBy;

```

Comment:
Why is this now not readonly?

Reply:
We need to set FilterBy from EmailDelegatesController Index POST, in the case of returning the view when there is an error. (We can't construct a new viewmodel there because we need to retain model state).

Reply:
Nevermind, we can just construct a new viewmodel and set the selected ids and delivery date separately.